File: sqs-playground/consumer.js
var AWS= require('aws-sdk');

AWS.config.region = 'eu-central-1'
var sqs = new AWS.SQS();

QUEUE_URL = "https://sqs.eu-central-1.amazonaws.com/862127070214/wisniewskit_tweets"

sqs.sendMessage({
    MessageBody: 'Hello World from JS',
    QueueUrl: QUEUE_URL
}, (err, data) => {
    if (err) console.log(err, err.stack)
    else console.log(data)
})

File: sqs-playground/consumer.py
import boto3
import time
import random

sqs = boto3.resource('sqs', region_name="eu-central-1")
QUEUE_URL = "https://sqs.eu-central-1.amazonaws.com/862127070214/wisniewskit_tweets"

tweets = sqs.Queue(QUEUE_URL)

while True:
    for message in tweets.receive_messages(WaitTimeSeconds=10):
        time.sleep(2)
        if random.randint(0,10) <= 5:
            print('Something went wrong')
            continue
        print("Message body: {}".format(message.body))
        message.delete()
    print('next iteration.....')